WalletWasabi.Gui/ViewModels/ViewModelBase.cs

```diff
@@ -1,50 +1,53 @@
-using Newtonsoft.Json;
 using ReactiveUI;
 using System;
 using System.Collections;
-using System.Collections.Generic;
 using System.ComponentModel;
-using System.Linq;
-using System.Reflection;
-using WalletWasabi.Gui.ViewModels.Validation;
-using WalletWasabi.Models;
+using WalletWasabi.Gui.Validation;
 
 namespace WalletWasabi.Gui.ViewModels
 {
-	public class ViewModelBase : ReactiveObject, INotifyDataErrorInfo
+	public class ViewModelBase : ReactiveObject, INotifyDataErrorInfo, IRegisterValidationMethod
 	{
+		private Validations _validations;
```

Comment:
Don't forget that we have a coding convention to use properties instead of fields - event if it is private.

Just a few comments why, https://stackoverflow.com/questions/2374416/why-use-simple-properties-instead-of-fields-in-c

Reply:
https://csharpindepth.com/articles/PropertiesMatter

Reply:
yes I will change it, but I think you have miss understood those articles...

properties matter for providing a public interface... but when this is implementation detail fields is valuable.

Take a look at .net core source, wpf source, avalonia source,they all use fields for implementation details... properties to manage the binary interface (public / private)

these are not part of the binary interface. so it doesnt make sense, but because the rules are there we can follow them.

Reply:
Ok iv done it for the other 2... but not for this one... 
see line 22... this is a property already.

`protected IValidations Validations => _validations;`

Reply:
Note, internally this is accessed via the field to get the full api of that class, and the property exposes just the IValidations interface... this has to be one of the best examples of where this coding standard rule wouldnt make sense...

Reply:
Why are you against making everything property? Make you rational about that. 

I made my point: it is our coding convention.

In this specific case, I accept your reasoning, however it could be easily solved by renaming.